### PR TITLE
re-enable disabled tests

### PR DESCRIPTION
**Pull request by dansiegel**
GitHub Issue (If applicable): closes #

- n/a

## PR Type

What kind of change does this PR introduce?

- Build or CI related changes

### CODE REVIEW

File: build/.azure-pipelines.TemplateValidation.yml [hunk 1]
```diff
@@ -79,17 +79,15 @@ jobs:
...
-      # MauiRecommended:
-      #  templateArgs: '-preset=recommended -maui'
+      MauiBlank:
+       templateArgs: '-preset=blank -maui'
+      MauiRecommended:
+       templateArgs: '-preset=recommended -maui'
       ThemeService:
        templateArgs: '-theme-service'
-      # NOTE: Temporarily disabling so we can merge #585
       # https://github.com/unoplatform/uno.templates/issues/396
```
Comment (agneszitte) [inline build/.azure-pipelines.TemplateValidation.yml:88]: Do we need to keep this as #396 is closed?

File: build/.azure-pipelines.TemplateValidation.yml [hunk 2]
```diff
@@ -79,17 +79,15 @@ jobs:
...
-      # Issue396:
-      #  templateArgs: '-preset blank -platforms windows -platforms android -presentation none -theme fluent -nav blank -log none -maui -markup csharp'
+      Issue396:
+       templateArgs: '-preset blank -platforms windows -platforms android -presentation none -theme fluent -nav blank -log none -maui -markup csharp'
       # https://github.com/unoplatform/uno.templates/issues/414
```
Comment (agneszitte) [inline build/.azure-pipelines.TemplateValidation.yml:91]: Do we need to keep this as #414  is closed?

File: build/.azure-pipelines.TemplateValidation.yml (hunk 1 above)
Comment (dansiegel) [reply build/.azure-pipelines.TemplateValidation.yml:88]: Yes these tests are there to ensure we do not have regressions

**Review by agneszitte — APPROVED** [review]

**Comment by dansiegel** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**